src/Tests/SNIPPETS.cs

```diff
@@ -54,6 +54,23 @@ public static class SNIPPETS
     }
 ";
 
+        public static string Op3_Op4_EntryPoints =
+@"
+    /// # Summary
+    ///     This to show that @EntryPoint() is ignored when compiling from snippets:
```

Comment:
I can't really see but did you try white spaces before and after?

Reply:
I will update this to include a more thorough test of the regex, e.g. `@ EntryPoint (  )` in addition to `@EntryPoint()`.